lib/options.js

```diff
@@ -50,22 +50,23 @@ function compute(processArgv) {
     processArgv.slice(2).includes('--help') ||
     processArgv.slice(2).includes('-h');
 
-  /** @type {ParsedArgs} */
-  const args = minimist(processArgv.slice(2), {
-    alias: Flags.flags.reduce((object, flag) => {
-      if (flag.alias !== undefined) {
-        object[flag.name] = flag.alias;
-      }
+  const args = /** @type {ParsedArgs} */ (
```

Comment:
What's the difference between
```
const args = /** @type {ParsedArgs} */ ...
```
and
```
/** @type {ParsedArgs} */
const args = ...
```
Are the two different from TS' point of view?
(Or is this just a styling change?)

Reply:
The first one is a cast, the second one is a type annotation. I don't recall why this needed a cast.